doc/tutorial/index.rst

```diff
@@ -161,28 +160,61 @@ us that the first packet indeed contains a Read Request:
     1367411052.081790 IP 192.168.0.253.50618 > 192.168.0.10.3445: UDP, length 4
     [...]
 
-Running Zeek on the trace with the ``udp-contents`` scripts produces
-the expected content files:
+We now run Zeek on that trace to perform the batch conversion:
 
 .. code::
 
-    # zeek -r tftp_rrq.pcap udp-contents
-    # ls udp-contents.orig.*
-    udp-contents.orig.1367411051.972852.dat
-    udp-contents.orig.1367411052.077243.dat
-    udp-contents.orig.1367411052.086300.dat
-    udp-contents.orig.1367411052.088995.dat
-    udp-contents.orig.1367411052.091675.dat
-    [...]
+    # zeek -r tftp_rrq.pcap policy/frameworks/spicy/record-spicy-batch SpicyBatch::filename=tftp_rrq.dat
+    tracking [orig_h=192.168.0.253, orig_p=50618/udp, resp_h=192.168.0.10, resp_p=69/udp]
+    tracking [orig_h=192.168.0.10, orig_p=3445/udp, resp_h=192.168.0.253, resp_p=50618/udp]
+    recorded 2 sessions total
+    output in tftp_rrq.dat
+
+This leaves a new ``spicy-driver`` batch file in ``tftp_rrq.dat``.
+(If we had left off, the ``SpicyBatch::filename`` argument, the
+default output name is ``batch.dat``.)
```

Comment:
Fixing up punctuation:

> This leaves a new ``spicy-driver`` batch file in ``tftp_rrq.dat``
(if we had left off the ``SpicyBatch::filename`` argument, the
default output name is ``batch.dat``).